app/templates/_proposition_header.html

```diff
@@ -4,7 +4,11 @@
     <nav id="proposition-menu">
       <a href="/" id="proposition-name">Digital Marketplace</a>
       <ul id="proposition-links">
-        <li><a class="home" href="/suppliers">Supplier login</a></li>
+        {% if current_user.email_address %}
+        <li><a class="home" href="{{ url_for('main.logout') }}">Log out</a></li>
+        {% else %}
+        <li><a class="home" href="{{ url_for('main.render_login') }}">Log in</a></li>
+        {% endif %}
         <li><a href="/suppliers/create">Create supplier account</a></li>
```

Comment:
Not yet I don't think - Ralph is working on the design still, but I think the link to "Create buyer account" will come as part of the create-a-brief frontend flow.